.gitignore

```diff
@@ -31,7 +31,8 @@
 *.out
 *.app
 
-build
+build*
+build_*
```

Comment:
Why ?

Reply:
Because I have multiple build folder for each branch I want to test

Reply:
Ok now after some reflection and usage, I'm ok with you I will delete it